deploy/cluster_role_binding.yaml

```diff
@@ -1,11 +1,12 @@
-kind: RoleBinding
+kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: ocs-operator
 subjects:
 - kind: ServiceAccount
   name: ocs-operator
+  namespace: default
```

Comment:
remove namespace. `ocs-operator` sa could be in any namespace.

Reply:
no we can't remove the namespace as the serviceAccount is namespaced resource. But the namespace must be openshift-storage.